container-build/cerc-fixturenet-eth-geth/genesis/el/el-config.yaml

```diff
@@ -0,0 +1,21 @@
+mnemonic: "viable ketchup woman library opinion copy rhythm attend rose knock penalty practice photo bundle budget dentist enter round bind holiday useful arch danger lobster"
```

Comment:
Should this be in ./config?

Reply:
Same as above, it is part of the input used to generate the genesis during the container build.

Reply:
I think I got confused between build_el.sh and run_el.sh